app/views/contexts/summary/group.py

```diff
@@ -28,9 +28,15 @@ def __init__(
         progress_store: ProgressStore,
         return_to: str | None,
         return_to_block_id: str | None = None,
+        summary_type: str | None = None,
+        view_submitted_response: bool | None = False,
     ) -> None:
         self.id = group_schema["id"]
-        self.title = group_schema.get("title")
+
+        if summary_type == "Summary" and not group_schema.get("title"):
+            self.title = schema.get_title_for_section(location.section_id)
+        else:
+            self.title = group_schema.get("title")
```

Comment:
What if it's not summary_type "Summary" and no group title (`summary_type != "Summary" and not group_schema.get("title")`? if it's calculated summary as `schemas/test/en/test_new_calculated_summary_repeating_section.json`?

Reply:
I only added this as the Summaries looked slightly odd when displaying list collectors when there was no group title, as per the `berd` schema.  Calc summaries should stay the same

Reply:
<img width="472" alt="same_title" src="https://user-images.githubusercontent.com/114991656/234819962-e5b65b2e-bd71-41c0-a3d3-c18e6436d8dc.png">
Having entered "Person a", "Person b", and "Person c" in the household section, it'd be nice to see those names on the summary, rather than the group title for each one, as currently I lose which item corresponds to each person in the list. I don't know if thats possible to pipe those in (or if in typical use it doesn't matter) but I think it looks slightly odd in this particular test

Reply:
It would be possible, but we'd need to allow group titles to support placeholders which they do not at the moment. This is what I was doing initially but I realised that the schema was failing validation, and it prompted me to raise this question is slack: https://surveydatacollection.slack.com/archives/CLR17KQKU/p1681906866743749

Reply:
This issue has been resolved but the rendering issue will be fixed in a follow up PR.